retrieveProducts/scripts/dml.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO product(title, description, price, img)
+VALUES('Pixel 5', 'Pixel is built the Google way, so it keeps getting more helpful with feature drops.5 New features and apps are frequently added through software boosts so your Pixel gets better over time.', 599, 'https://www.ixbt.com/img/n1/news/2020/9/4/gsmarena_003-1-3_large.jpg'),
+      ('IPhone 12', 'Blast past fast. 5G speed. A14 Bionic, the fastest chip in a smartphone. An edge-to-edge OLED display. Ceramic Shield with four times better drop performance. And Night mode on every camera. iPhone 12 has it all — in two perfect sizes.', 699, 'https://specials-images.forbesimg.com/imageserve/5eebf1a63f00c50007f33477/960x0.jpg?fit=scale'),
+      ('OnePlus 8', 'The OnePlus 8 has a 6.55-inch AMOLED panel with sloping sides, a wide colour gamut, and a hole-punch cutout in the upper left corner. The 90Hz refresh rate makes usage in general feel fluid and snappy. It also has an in-display fingerprint sensor, which is super quick at authentication.', 599, 'https://5zv.by/image/cache/foto_edit/OnePlus_8_1-1000x1000.jpg'),
+      ('OnePlus 8 Pro', 'The OnePlus 8 Pro is an IP68 rated phone that is resistant to dust and water. It comes with an impressive 6.78-inch AMOLED panel that has a resolution of 1440x3168 pixels with high color accuracy and HDR10+ support. Also, it can run at 120Hz at the QHD+ resolution. The display of the.', 999, 'https://shop.by/images/oneplus_8_pro_8gb_128gb_blue_(evropeyskaya_versiya)_1.jpg'),
+      ('OnePlus 8T', 'The OnePlus 8 has a 6.55-inch AMOLED panel with sloping sides, a wide colour gamut, and a hole-punch cutout in the upper left corner. The 90Hz refresh rate makes usage in general feel fluid and snappy. It also has an in-display fingerprint sensor, which is super quick at authentication.', 699, 'https://www.ixbt.com/img/n1/news/2020/8/3/image-111-954x1024_large_large.png'),
+      ('IPhone 11', 'Apple on September 10, 2019, unveiled the iPhone 11, its flagship $699 smartphone that offers a range of powerful features at an affordable price tag. Sold alongside the more expensive iPhone 11 Pro and the iPhone 11 Pro Max, the iPhone 11 is the iPhone that''s ideal for most people.', 799, 'https://cdn1.it4profit.com/AfrOrF3gWeDA6VOlDG4TzxMv39O7MXnF4CXpKUwGqRM/resize:fit:670/bg:f6f6f6/q:100/plain/s3://catalog-products/190911080934070763/190916160018960154.png@jpeg')
+
+
+INSERT INTO store(product_id, count) 
+VALUES  ('5202bc13-2e2a-4d66-979b-c457a31de274', 3),
```

Comment:
These values are auto-generated and next run they will differ, so it won't work

Reply:
This is just initial script to populate data. It was run manually and you do not need to run it each time

Reply:
You don't know in advance if you'll have to to reuse it for another environment. Using just a small modification you can make it flexible enough to be reused